web/public/mgmt/config.yml

```diff
@@ -3060,6 +3066,22 @@ collections:
       preview: false
     delete: false
     files:
+      - label: "Quick Action Icons"
+        name: "arrayOfQuickActionIcons"
+        file: "content/src/mappings/quickActionIcons.json"
+        fields:
+          - label: Quick Action Icons
+            name: arrayOfQuickActionIcons
+            widget: list
+            allow_add: false
+            summary: "{{name}}"
+            fields:
+              - label: Name
+                name: name
+                widget: string
```

Comment:
dust - I don't think this should be editable by Content, if a change is made, the icons won't render.

Reply:
nvm, I see there is a name and id/filename field. If a change is made, I think the files with the previous value would not render. Is content responsible for making updates. I kind of lean on making it hidden

Reply:
I think it would be fine actually, the name can be edited so long as the ID stays the same, as the files only have ID's and that's what is being used for reference I believe. It just allows content to change the name that corresponds to the value of the file. 

Like the ID is the only thing stored in the files and the name is just for display in the CMS so the name can be changed without ruining anything I believe.